containers/compilers/osv/java-compiler/java-main-caller/src/main/java/com/emc/wrapper/UDPBootstrap.java

```diff
@@ -1,160 +0,0 @@
-package com.emc.wrapper;
```

Comment:
Hm, I totally forgot that for VirtualBox we need Wrapper that answers to UDP broadcasting. As @gberginc said, lets leave OSv Java compiler untouched for now. I've reverted all the deletions.